Edit new joiners guide (#46)
alan-turing-institute/REG-handbook: content/docs/onboarding/new_joiners/systems_set_up.md

### Create a Public Profile on the Turing Website

> Get in touch with the [Web team](mailto:website@turing.ac.uk).

Reviewer: Can you add the web team to '[The REGistry](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry#responsibilities)' under the 'Points of Contact' section and link to that?

We try to consolidate that information and keep emails/identifies off the public site.

Author: Done

> Send them an email with your bio and a profile picture attached.

Reviewer: ```suggestion
Send them an email with your biography and a profile picture attached.
```

[Mathison](https://mathison.turing.ac.uk/) is the Institute's intranet. A few things still need to be done on the old intranet, [Turing Complete](https://turingcomplete.topdesk.net).

### Purchasing home office equipment

> There is a budget to purchase peripherals (monitor, mouse, keyboard, etc.) as well as other equipment such as an office chair and desk.

Reviewer: ```suggestion
There is a budget to purchase peripherals (monitor, mouse, keyboard, _etc._) as well as other equipment such as an office chair and desk.
```

> For peripherals, get in touch with [IT Services](mailto:ITServices@turing.ac.uk).

Reviewer: ```suggestion
For peripherals, get in touch with [IT Services]({{< ref "https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry#points-of-contact" >}}).
```
